DigitalLearningSolutions.Web/ViewModels/MyAccount/MyAccountEditDetailsFormData.cs

```diff
@@ -62,5 +77,84 @@ protected MyAccountEditDetailsFormData(MyAccountEditDetailsFormData formData)
         public IFormFile? ProfileImageFile { get; set; }
 
         public bool IsDelegateUser { get; set; }
+
+        [RunValidateMethod(nameof(ValidateProfessionalRegistrationNumber))]
+        public string? ProfessionalRegistrationNumber { get; set; }
+
+        [RunValidateMethod(nameof(ValidateHasProfessionalRegistrationNumber))]
+        public YesNoSelectionEnum HasProfessionalRegistrationNumber { get; set; }

```

Comment:
Do you think it's better to use `YesNoSelectionEnum` than `bool?`?
I think the general use case for radio buttons will be "some list of values or null", so having a nullable type for this kind of data makes sense to me (if c# casts '' to null nicely).

Reply:
So this is part of the issue I raised on slack regarding the whole approach to the radio buttons - this hand rolled solution here although simpler steps away from what is done elsewhere.

Reply:
Specifically on the bool option. I've gone for that. In this context it works fine.